lpc55_sign/src/verify.rs

```diff
@@ -3,16 +3,22 @@
 // file, You can obtain one at https://mozilla.org/MPL/2.0/.
 
 use crate::{cert, Error};
+use der::Encode as _;
 use hex::ToHex as _;
-use log::{debug as okay, info, trace, warn};
+use log::{debug as okay, error, info, trace, warn};
```

Comment:
Why are we importing `error` here?

We deliberately declared a separate `error!` macro below, which takes a boolean `failed` flag as its first argument.  This protects us from accidentally failing to set the failed flag when an error occurs.

Reply:
I think this was a mis-merge on my part, thanks. Fixed in fcc94b5.